src/app/store/submitted-transactions/submitted-transactions.selectors.ts

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from '@reduxjs/toolkit';
+import { useSelector } from 'react-redux';
+
+import { RootState } from '@app/store';
+
+const selectSubmittedTransactionsSlice = (state: RootState) => state.submittedTransactions;
+
+export const selectSubmittedTransactions = createSelector(
+  selectSubmittedTransactionsSlice,
+  state => state.entities
+);
```

Comment:
Can use submitedTxAdapter to generate this automatically

Reply:
Do you know why you did it this way for the key selector?

Reply:
I do not, this could also use `keyAdapter.getSelectors().selectById`

Reply:
What is best in your opinion, to use `selectEntities` and use the dictionary (which is how the code is now), or `selectAll` and use an array of entities.

Reply:
Afaict, you're only ever consuming it as an array, so better to export as an array `selectAll`. 

If, at some point, we need to find a specific one by id, I'd create another higher order selector that takes an id param (sooner than using `array.find`)